sdks/wasm/Makefile

```diff
@@ -324,6 +327,43 @@ $(eval $(call XunitTestTemplate,System.Xml-xunit,System.Xml-xunit,wasm_System.Xm
 $(eval $(call XunitTestTemplate,System.Xml.Linq-xunit,System.Xml.Linq-xunit,wasm_System.Xml.Linq_xunit-test.dll))
 $(eval $(call XunitTestTemplate,corlib-xunit,corlib-xunit,wasm_corlib_xunit-test.dll))
 
+#
+# Targets for building time zone data
+#
+
+
+$(TZ_INPUT)/.stamp-tz-data:
+	mkdir -p $(TZ_INPUT)
+	mkdir -p $(TZ_OUTPUT)
+	curl -L https://data.iana.org/time-zones/tzdata-latest.tar.gz -o $(TZ_INPUT)/tzdata.tar.gz
+	tar xvzf $(TZ_INPUT)/tzdata.tar.gz -C $(TZ_INPUT)
+	touch $@
```

Comment:
Is it possible to do this correctly incrementally? I suspect not.
You know, like, only go to the network when needed? Neither too much nor too little?
Can you use curl to get just a checksum first?

Reply:
Its 400k.

Reply:
Has been removed in favor of `mono_webassembly_zoneinfo.js` module